tests/storage_test.go

```diff
@@ -496,69 +455,70 @@ var _ = Describe("[Serial]Storage", func() {
 						}, 200)).To(Succeed())
 					}
 
-					err = virtClient.VirtualMachineInstance(vmi.Namespace).Delete(vmi.Name, &metav1.DeleteOptions{})
+					err = virtClient.VirtualMachineInstance(obj.Namespace).Delete(obj.Name, &metav1.DeleteOptions{})
 					Expect(err).ToNot(HaveOccurred())
-
-					tests.WaitForVirtualMachineToDisappearWithTimeout(obj, 120)
+					Eventually(ThisVMI(obj), 120).Should(BeGone())
 				}
 			})
 		})
 
+		Context("[Serial]Without feature gates disabled for", func() {
```

Comment:
Is it Without or With feature gates disabled?

Reply:
Fixed.